randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -2160,7 +2160,7 @@ Extra - asset_id: collision_camera_025_B
   > Door to Teleport to Burenia (Power Beam)
       Any of the following:
           Speed Booster or Use Spin Boost
-          Infinite Bomb Jump (Intermediate) and Lay Bomb
+          Infinite Bomb Jump (Advanced) and Lay Bomb
```

Comment:
Intermediate for Cross, Advanced for normal

Reply:
I'm not sure how this can be done with Cross, so I make this Normal Bombs only.

Reply:
This is the Ghavoran one at the Storm Gate right? You can break the enky platform with a well-placed cross bomb, then just ibj up normally.

Reply:
Oh, and i think speed can't be charged here right? Might want to take that out too.

Reply:
Interesting, I tried to do that, but I must not know where to place the cross bomb for that. In that case, would that enable using simple ibj after having placed that cross bomb?

Reply:
The speed booster usage is something I didn't intend to focus on in this PR, but I could remove it. Alternatively, I could make this be the first usage of Short Boost, which already has a trick and a template.

Reply:
> Interesting, I tried to do that, but I must not know where to place the cross bomb for that. In that case, would that enable using simple ibj after having placed that cross bomb?

It would yes. as for the speed, I'd let Izzy and Merry handle that, since they added the trick.

Reply:
For what it's worth you dont need short boost if you get the speed form the previous room, but i dont know how to handle that well at this moment

Reply:
I ended up taking out the shinespark, but leaving a todo comment. I also added using cross bomb into Simple IBJ, which I don't know if should be behind a trick, maybe Knowledge at some level?